### PR TITLE
UI/jade feedback

### PR DESCRIPTION
**Pull request by jmzwar**
Mainly changes to action cards but includes Settings

### CODE REVIEW

File: v2/components/MainActionCards/MainActionCards.tsx [hunk 1]
```diff
@@ -201,83 +216,84 @@ const CollectActionCard: React.FC<{
...
     liquidationCratioPercentage,
     targetCratioPercentage,
     currentCRatioPercentage,
   });
-  const isStaking = currentCRatioPercentage > 0;
+
+  const isStaking = currentCRatioPercentage && currentCRatioPercentage > 0;
   const canClaim = !hasClaimed && variant === 'success';
+
+  console.log('is loading', isLoading);
```
Comment (bachstatter) [inline v2/components/MainActionCards/MainActionCards.tsx:230]: missed log

File: v2/components/Settings/Settings.tsx [hunk 2]
```diff
@@ -0,0 +1,158 @@
+import {
```
Comment (bachstatter) [inline v2/components/Settings/Settings.tsx:1]: This is currently not used? I think we can release the first version without this

File: v2/components/MainActionCards/MainActionCards.tsx [hunk 3]
```diff
@@ -1,198 +1,213 @@
...
+  icon: ReactNode;
+  disabled: boolean;
+  Content: JSX.Element | null;
+  buttonVariant?: 'solid' | 'outline' | 'link';
+  buttonText: string;
+  buttonAction: () => void;
+  testId: string;
+}
+
+export const Card = ({
```
Comment (bachstatter) [inline v2/components/MainActionCards/MainActionCards.tsx:34]: This is a nicer abstraction 👍

File: v2/components/Settings/Settings.tsx (hunk 2 above)
Comment (jmzwar) [reply v2/components/Settings/Settings.tsx:1]: Yeah - I added it but I think we can release without implementing it in ui

**Review by bachstatter — APPROVED** [review]
Remove some logs, fix CI and then good to go

File: v2/components/MainActionCards/MainActionCards.tsx [hunk 4]
```diff
@@ -1,198 +1,213 @@
...
+      icon={<StakeIcon disabled={isLoading} />}
+      disabled={isLoading}
+      buttonText={
+        isStaking
+          ? t('staking-v2.main-action-cards.stake-link-button')
+          : t('staking-v2.main-action-cards.stake-main-button')
+      }
+      Content={null}
+      buttonVariant={isStaking ? 'link' : isLoading ? 'link' : 'solid'}
+      testId={isStaking ? 'main stake button' : 'start staking button'}
```
[the remark this answers is not in the document]
Comment (jmzwar) [reply v2/components/MainActionCards/MainActionCards.tsx:116]: We can put it directly on the card - but it'll still be a ternary depending on isStaking

**Comment by bachstatter** [comment]
> The abstraction of card feels a bit too early in my opinion and we could do with some copy paste for a while. Testid that is passed down through props is a clear indication to me that abstraction is not yet formed to be "abstract enough", this will be more painful to support. But this is obviously not a blocker for feature if time constrained

I think James was already feeling some pain with previous abstractions. It was hard to just add a disabled prop. ( the reason for why he did the card abstraction)

**Pull request merged**